CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.12.1]
+
+### Fixed
+
+- [#5260](https://github.com/firecracker-microvm/firecracker/pull/5260): Fixed a
```

Comment:
we should probably quote this PR (5277), not the one from the main

Reply:
fixed